lib/tasks/uri_cache.rake

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+namespace :hyku do
```

Comment:
Just an idea, should we connect this to a frontend button to allow superadmin users run this themselves?

Reply:
Definitely a cool idea, a few question do come to mind though.  Where would we put the button?  Should the reindex rakes (since this is similar) be also be executed on the frontend?  Could we wait until the the clients request this feature?  We're extremely behind on this project so I'm weary of adding bells and whistles that weren't asked for.

Reply:
I was just thinking in practice what would this look like. How often do these terms change and how would we know if they did. So at the minimum if there's concern running the task as a cron job once every x months where it wouldn't involve a dev may be the happy medium. But I understand your concerns in our given situation.